### DISCUSSION TITLE
Need guidance in Survival analysis using RSF in scikit learn

### DISCUSSION THREAD

**Discussion by rasheed2063** [sebp/scikit-survival#323] [Q&A]
Hello friends,
I want to do survival analysis for testing of concrete columns.The test dataset includes applied load on columns and the failure status as shown in figures. I have loaded the dataset and split into X and Y but when i do the rsf, i get an error "y must be a structure array".

Any guidance will be appreciated.![image](https://user-images.githubusercontent.com/83563415/203478547-182120ac-5dd2-488a-a03a-9b8032b28e4e.png)
![image](https://user-images.githubusercontent.com/83563415/203478590-9cbcdaf2-f4a1-4da8-974d-eddd330a106c.png)
![image](https://user-images.githubusercontent.com/83563415/203478619-6f2f2f08-273f-4f42-a98f-b4f443ca4218.png)
![image](https://user-images.githubusercontent.com/83563415/203478662-a7f31c0f-1331-4a97-a705-8e1f0efe79a2.png)

**Comment by sebp** [comment]
I don't see any obvious mistake, maybe it's because the first field has an empty name (`''`)?

You can also use the helper class [sksurv.util.Surv](https://scikit-survival.readthedocs.io/en/latest/api/generated/sksurv.util.Surv.html) to create `y`.
